docs/guides/install.md

```diff
@@ -154,6 +154,7 @@ The interactive install will guide you through a series of questioned designed t
       <ul>
         <li>Monitor is only available for install if IoT is selected</li>
         <li>Assist and Predict are only available for install if Monitor is selected</li>
+        <li>From now on Assist will be labeled as Collaborate. Although while installing the Assist you will see Assist right now but after logging in you will find Collaborate in place of Assist. Soon this is also gonna be improved and while installing also you will be able to see Collaborate.</li>
```

Comment:
I am not sure this still reads right. I think it should be more like:
"From MAS 9.1 onwards, Assist will be rebranded as Collaborate in the MAS UI. It will still appear as Assist in the MAS CLI and within the OpenShift Cluster, but from the MAS UI it will appear as Collaborate."